app/views/thesis/proquest_export_preview.html.erb

```diff
@@ -0,0 +1,56 @@
+<%= content_for(:title, "Thesis Processing | MIT Libraries") %>
+
+<% content_for :additional_js do %>
+  <script src="https://cdn.datatables.net/1.11.3/js/jquery.dataTables.min.js"></script>
+<% end %>
+
+<link href="https://cdn.datatables.net/1.11.3/css/jquery.dataTables.min.css" rel="stylesheet">
+
+<h3 class="title title-page">Export theses for ProQuest</h3>
+
+<%= render 'shared/you_are' %>
+
+<div class="gridband" id="export-action" aria-live="polite" role="region">
+  <div class="inline-action well warning">
+    <% if @theses.count < 1 %>
+      <p>No theses are available to export.</p>
+    <% else %>
+      <div class="message">
+        <h4 class="title">Ready to export <%= @theses.count %> theses?</h4>
+        <p>If the set of thesis records below looks correct, click the button at right to export their handles. Use the
```

Comment:
Small wording suggestion: `If the set of thesis records below is correct...`. I think it may be worthwhile to be clearer that the accountability of the record set is on the person clicking `export`